docs/labs/format-strings.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta http-equiv="X-UA-Compatible" content="IE=edge">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<link rel="stylesheet" href="https://best.openssf.org/assets/css/style.css">
+<link rel="stylesheet" href="checker.css">
+<script src="js-yaml.min.js"></script>
+<script src="checker.js"></script>
+<link rel="license" href="https://creativecommons.org/licenses/by/4.0/">
+
+
+<script id="expected0" type="plain/text">
+    return '{format_string}, {event.level}'.format(format_string=user_input, event=event)
+</script>
+
+<!-- Full pattern of correct answer -->
+<script id="correct0" type="plain/text">
+\s*return\s'\{format_string\},\s\{event\.level\}'\.format\(format_string=user_input,\sevent=event\)\s*
+</script>
+
+<script id="info" type="application/yaml">
+---
+- absent: "user_input"
```

Comment:
You're missing this line after `---`:

~~~~
hints:
~~~~

We support several different keys in the info, "hints" is one of them.

Once you do, you'll find a lot of the tests fail. You'll need the tests to pass :-).